tools/linter_lib/custom_check.py

```diff
@@ -694,7 +694,7 @@
             "templates/corporate/billing.html",
             "templates/corporate/upgrade.html",
             # Miscellaneous violations to be cleaned up
-            "web/templates/user_info_popover_title.hbs",
+            "web/templates/user_card_popover_title.hbs",
```

Comment:
It would be very nice if the commit message here had the list of identifiers that you renamed, e.g. one per line, so that one can attempt to verify the changes in part be grepping for each of them before/after.

Reply:
Fixed in series of renaming commits.